tests/test_name_server.rb

```diff
@@ -44,8 +44,8 @@ def no_nameserver_google
   # TESTS
 
   def test_nameserver_create_destroy_single_ipv4
-    id = '8.8.8.8'
-    refute_includes(Cisco::NameServer.nameservers, id)
+    id = '7.7.7.7'
+    assert_equal({}, Cisco::NameServer.nameservers)
```

Comment:
This is why the tests questionably cover the "empty" case; either the tests tiptoe around pre-existing name-servers (note that this is non-vrf name servers only) and therefore don't test empty cases, or they brazenly assume this is a throwaway system (which in my case it is; it's vagrant) and `assert_empty()`. Which would you like?